src/sentry/incidents/subscription_processor.py

```diff
@@ -633,19 +635,65 @@ def trigger_resolve_threshold(
     def handle_trigger_actions(
         self, incident_triggers: List[IncidentTrigger], metric_value: float
     ) -> None:
-        actions = deduplicate_trigger_actions(triggers=deepcopy(incident_triggers))
+        actions = deduplicate_trigger_actions(triggers=deepcopy(self.triggers))
         # Grab the first trigger to get incident id (they are all the same)
         # All triggers should either be firing or resolving, so doesn't matter which we grab.
         incident_trigger = incident_triggers[0]
-        method = "fire" if incident_triggers[0].status == TriggerStatus.ACTIVE.value else "resolve"
+        method = "fire" if incident_trigger.status == TriggerStatus.ACTIVE.value else "resolve"
+        try:
+            incident = Incident.objects.get(id=incident_trigger.incident_id)
+        except Incident.DoesNotExist:
+            # TODO: increment metric??
```

Comment:
When we do this in `handle_trigger_action`, we have a metric increment here. Should I do the same here?  Should I create a new metric? 

Wasn't sure if it made sense since that metric appears to be related to trigger actions and not all actions together like we are here. Here's the other place I'm talking about:

https://github.com/getsentry/sentry/blob/471b6d36f5a3558d42115528e9723c80f16deeab/src/sentry/incidents/tasks.py#L192-L196C15

Reply:
We use these metrics to monitor and alert on features via Datadog. I think we can increment the same metric here as well since we hit this `except` case frequently.